### PR TITLE
environment_check.sh: check the hostname uniqueness

### PR DESCRIPTION
**Pull request by derekbit**
longhorn/longhorn#5012

Signed-off-by: Derek Su <derek.su@suse.com>

### CODE REVIEW

File: scripts/environment_check.sh [hunk 1]
```diff
@@ -233,70 +242,101 @@ check_package_installed() {
...
     error "Please install missing packages."
     exit 2
-  else
-    info "Required packages are installed."
   fi
+
+  info "Required packages are installed."
+}
+
+check_hostname_uniqueness() {
```
Comment (innobead) [inline scripts/environment_check.sh:260]: Why not just check the K8s node resource? there should be hostname in the status?

ref: https://kubernetes.io/docs/concepts/architecture/nodes/#addresses

**Review by innobead — CHANGES_REQUESTED** [review]

File: scripts/environment_check.sh (hunk 1 above)
Comment (innobead) [reply scripts/environment_check.sh:260]: But the change LGTM.

Comment (derekbit) [reply scripts/environment_check.sh:260]: Ah, right. Checking the node resources is better.

Comment (derekbit) [reply scripts/environment_check.sh:260]: Updated. Thank you.

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
cc @longhorn/qa

**Pull request merged**